iidm/iidm-api/src/main/java/com/powsybl/iidm/network/CurrentLimits.java

```diff
@@ -90,6 +90,10 @@
      */
     TemporaryLimit getTemporaryLimit(int acceptableDuration);
 
+    default CurrentLimits setTemporaryLimit(String name, double value, int acceptableDuration, boolean fictitious) {
```

Comment:
I think we should have several methods for replace name acceptable duration fictitious and value (with method chaining). One method with several parameters is not readable and it is the design chosen all over iidm api

Reply:
I created a `TemporaryLimitUpdater` implementing `TemporaryLimitAdder` to do this. It needed a few restructuration of the `CurrentLimitsAdder` and `CurrentLimits` API and implementations.